### DISCUSSION TITLE
Security Onion ISO 2.4

### DISCUSSION THREAD

**Discussion by tslvr3** [Security-Onion-Solutions/securityonion#11320] [2.4] [answered]
Hello Team,

I have tried to download the security Onion 2.4 image multiple times but was not able to find the link.

Please help me download it.

Thanks & Regards
TSLVR

**Comment by Mav1814** [comment, marked as answer]
Hello, 

This is the link where you can download the iso. 

[https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md](url)

And here is the link for the iso

[https://download.securityonion.net/file/securityonion/securityonion-2.4.10-20230821.iso](url)

**Reply by tslvr3** [reply]
Thank you so much :) 👍
